### PR TITLE
Normalize white-space → whitespace, semi-colon → semicolon

### PR DESCRIPTION
**Pull request by clintval**
I navigate documentation with <kbd>Ctrl</kbd> + <kbd>F</kbd> or <kbd>⌘ Cmd</kbd> + <kbd>F</kbd> frequently and found it difficult to understand when whitespace was, or was not, allowed in various parts of these specifications. Normalizing the terms helps search. If contributions like this are welcome, I could make more in the future, but do realize they're small in impact.

Before this PR:

| Term        | Count |
| :---        | ---   |
| white-space | 18    |
| whitespace  | 10    |
| Whitespace  | 0     |
| White-space | 0     |
| semi-colon  | 16    |
| semicolon   | 15    |
| Semicolon   | 2     |
| Semi-colon  | 4     |

After this PR:

| Term        | Count |
| :---        | ---   |
| white-space | 0     |
| whitespace  | 28    |
| Whitespace  | 0     |
| White-space | 0     |
| semi-colon  | 0     |
| semicolon   | 27    |
| Semicolon   | 6     |
| Semi-colon  | 0     |

### CODE REVIEW

**Comment by jkbonfield** [comment]
Small changes are fine and most welcome too.  Thank you.

I'll let the VCF spec maintainers decide on this obviously, but for what it's worth the use in SAM is "whitespace" and "semicolon".

File: VCFv4.1.tex
Comment (jmarshall) [inline VCFv4.1.tex:162]: While this line is being changed, it could be changed to `Semicolon-separated` (it's already hyphenated thus elsewhere in _VCF*.tex_). And similarly on the `\item FT` line below; and both of these ×4 _VCF*.tex_ files.

**Comment by jmarshall** [comment]
In the SAM documents, we've standardised on the ‘one word without hyphens’ forms as those are the canonical variants used by Wikipedia — [whitespace](https://en.wikipedia.org/wiki/Whitespace_character) and [semicolon](https://en.wikipedia.org/wiki/Semicolon) — among other reasons. Good point about consistent usage being important for  <kbd>⌘ Cmd</kbd> + <kbd>F</kbd> searching.

**Comment by clintval** [comment]
I was surprised to find another wordsmithing pedant in this PR! 👋

**Review by tskir — APPROVED** [review]
@clintval Thank you for this contribution! Stylistic changes are very much appreciated.

**Pull request merged**
